uim/uim
# Add support for Katakana-mode to in T-Code

Generally, Katakana-mode isn't needed in T-Code.
Because T-Code provides direct 2-strokes set for Katakana.
Users can use them to input Katakana.

If we add support for Katakana-mode, users just remember 2-strokes set for Hiragana to input Katakana.

## How to reproduce
1. set code table file of tut-code to `tcode.scm`
1. restart uim
1. switch input method to tut-code
1. switch to katakana mode
1. type `yf`
1. `あ` will appear (this is Hiragana, but Katakana `ア` is expected)
